<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RomanGUIClass</class>
 <widget class="QWidget" name="RomanGUIClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RomanGUI</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Burbank Big Cd Bd</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 27px &quot;Burbank Big Cd Bd&quot;;</string>
   </property>
   <property name="text">
    <string>Marv's Epic Roman Numeral Calculator</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="num1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="opSelection">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="num2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Roman Numeral 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Roman Numeral 2</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Num1_out">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>211</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 204, 255);</string>
   </property>
   <property name="title">
    <string>Number 1</string>
   </property>
   <widget class="QLineEdit" name="ArabicNum1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arabic Representation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RomanNum1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Roman Representation</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Num2_out">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>211</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 204, 255);</string>
   </property>
   <property name="title">
    <string>Number 2</string>
   </property>
   <widget class="QLineEdit" name="ArabicNum2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arabic Representation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RomanNum2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Roman Representation</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Num3_out">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>211</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 193, 149);</string>
   </property>
   <property name="title">
    <string>Result</string>
   </property>
   <widget class="QLineEdit" name="ArabicResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arabic Representation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RomanResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Roman Representation</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>I Solemnly swear not to try and break this program</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Submit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RomanGUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>num1</sender>
   <signal>textEdited(QString)</signal>
   <receiver>RomanNum1</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
